src/lib.rs

```diff
@@ -1,7 +1,16 @@
+//! Welcome to the code-documentation of flakeshot!
```

Comment:
I don't think we can consider this file as a general entry point every reader will start with

Reply:
Why not? It's the first page which pops up if you open the docs.

Reply:
Huh? Which page?

Reply:
just execute `cargo doc --open`

Reply:
Ha, didn't knew that :)